changes/571.fixed

```diff
@@ -0,0 +1 @@
+Increased Change Control URL field limit from 256 to 2048 characters in Config Plans and Bulk Edit.
```

Comment:
- Move `Increased Change Control URL field max length from 256 to 2048 characters in Config Plan model.` to a `994.changed`
- Rename this file to `994.fixed`

```suggestion
Fixed Change Control URL field max length in the Config Plan bulk edit form.
```